packages/mint-components/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.2] - 2021-08-20
```

Comment:
Need to update to `1.2.0` since this adds some new functionality. (see https://semver.org/)